scripts/celery-and-api.sh

```
@@ -1,6 +1,6 @@
-nohup celery -A celery_app beat --loglevel=info 2>&1 &
+nohup python3.11 -m celery -A celery_app beat --loglevel=info 2>&1 &
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)